content/SCALE/SCALECLIReference/Auth/CLITwoFactor.md

```diff
@@ -12,4 +12,243 @@ tags:
+
+Where:
+* *true/false* enables (true) or disables (false) 2FA.
+
+{{< expand "Command Example" "v" >}}
+```
+auth two_factor update enabled=true
+
+```
+{{< /expand >}}
+
+{{< expand "Setting the One-Time Password (OTP) Digit Amount">}}
+
+#### Description
+The `update otp_digits` command requires you to include between 6 and 8.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful, and returns an error when you enter an invalid integer.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth two_factor update otp_digits=<i>number</i></code>
+
+Where:
+* *number* is the number of digits between 6 and 8.
+
+{{< expand "Command Example" "v" >}}
+```
+auth two_factor update otp_digits=6
+
+```
+{{< /expand >}}
+
+{{< expand "Setting the Password Validity Window">}}
+
+#### Description
+The `update window` command extends the validity of one-time passwords, and requires you to include an integer.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful, and returns an error when you enter an invalid integer.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth two_factor update window=<i>number</i></code>
+
+Where:
+* *number* is the number of passwords before and after the current one that are still valid. Must be between 0 and 999999999999999999.
+
+{{< expand "Command Example" "v" >}}
+```
+auth two_factor update window=1
+
+```
+{{< /expand >}}
+
+{{< expand "Setting the One-Time Password Lifespan">}}
+
+#### Description
+The `update interval` command sets the lifespans of one-time passwords, and requires you to include an integer.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful, and returns an error when you enter an invalid integer.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth two_factor update interval=<i>number</i></code>
+
+Where:
+* *number* is the number (in seconds) a OTP will last before expiring. Must be between 5 and 999999999999999999.
+
+{{< expand "Command Example" "v" >}}s
+```
+auth two_factor update interval=30
+
+```
+{{< /expand >}}
+
+{{< expand "Enabling 2FA for SSH Logins">}}
+
+#### Description
+The `update services` command enables or disables 2FA for SSH logins, and requires you to include an argument.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns nothing when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth two_factor update services={"ssh":<i>true/false</i></code>
+
+Where:
+* *true/false* enables (true) or disables (false) SSH 2FA authentication.
+{{< expand "Command Example" "v" >}}
+```
+auth two_factor update services={"ssh":true}
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Verify Command
+
+The `verify` command verifies whether or not a password is authenticated.
+
+{{< expand "Renewing the 2FA Secret">}}
+
+#### Description
+The `verify` command requires the `token` property.
```

Comment:
if this requires the token property, then the next line should tell them how to enter the property argument (we have standard text for this in the template .